website/docs/components/tag/partials/guidelines/guidelines.md

```diff
@@ -1,82 +1,55 @@
-## When to use
+## Usage
 
-- Use tags to indicate an object's categorization, i.e., for filtering.
+### When to use
 
-## When not to use
+Use tags to indicate an object's categorization, i.e., for filtering.
 
-- for static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)
-- For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview)
-
----
-
-## Anatomy
-
-![Tag anatomy](/assets/components/tag/tag-anatomy.png)
-
-#### Text
+### When not to use
 
-Required
-
-#### Icon
-
-Required for isDismissible
-
-#### Container
-
-Required
-
----
+- For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)
+- For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview)
 
-## Layout
+### Layout
 
 - The Tag component can be seen in several different layouts, but we commonly see them in a list or a gorup.
```

Comment:
The tag component can be seen in several different layouts, but we commonly see them in a list or a group.